tgui/packages/tgui/components/Input.js

```diff
@@ -28,8 +28,12 @@ export class Input extends Component {
     };
     this.handleFocus = e => {
       const { editing } = this.state;
+      const { onFocus } = this.props;
       if (!editing) {
         this.setEditing(true);
+        if (onFocus) {
+          onFocus(e);
+        }
```

Comment:
Some input boxes i just want to use for displaying data and wanted to trigger a byond ui (like color picker) to change the value.